frontend/public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
```

Comment:
this is the public index.html page which gets sent to every client the first time they navigate to your app, and this title will be the title they see in the tab of their browser. you're right that we want to change this from React App to something else, but it should probably be the name of your app instead of just the page you're working on!

there are programmatic ways to update the title later on

Reply:
Just changed it to SkillSwap, I forgot why I did that if I'm being honest lol.